style.css

```diff
@@ -1,7 +1,121 @@
-h1 {
-  font-family: 'Roboto', sans-serif;
+body {
+  margin: 0;
 }
 
-span {
-  font-family: 'PT Serif', serif;
+.wrapper {
+  margin: auto;
+  width: 80%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo, 
+.nav {
+  display: inline-block;
+}
+
+.logo {
+  position: relative;
+  padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px PT Serif, Arial, Helvetica, sans-serif;
+}
+
+.logo__circle {
+  width: 28px;
+  height: 28px;
+  position: absolute;
+  top: 50%;
+  left: 0;
+  transform: translateY(-50%);
+  background: #0056ec;
+  opacity: 0.7;
+  border-radius: 50%;
+}
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #c8dbfb;
+  opacity: 0.7;
+  border-radius: 50%;
+}
+
+/* .logo {
+  width: 190px;
+  height: 35px;
+  background: url("img/logo.svg");
+} */
+
+.nav {
+  display: inline-block;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  padding: 0 13px;
+  font-size: 18px;
+  font-family: Roboto;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__item:hover {
+  text-decoration: underline;
+  border-bottom: solid #0056ec;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+}
+
+.section {
+  justify-content: space-between;
+  display: flex;
+  height: calc(100vh - 118px);
+}
+
+.main {
+  width: 37%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.main__title {
+  font: bold 66px/85px PT Serif, Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding:0 0 20px 0;
+}
+
+.main__title-with-background {
+  background-color: #ecf3ff;
+}
+
+.main__text {
+  font-size: 22px;
+  font-family: Roboto;
+  padding: 20px 0 40px 0;
+  width: 80%;
+  margin: 0;
+}
+
+.main__button {
+  font-size: 18px;
+  font-family: Roboto;
+  background-color: #0056ec;
+  color: #FFFFFF;
```

Comment:
Codacy found an issue: [Expected "#FFFFFF" to be "#FFF" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/app.creation/pullRequest?prid=6206393)